Documentation/hotdoc/gen-hotdoc.sh

```diff
@@ -14,6 +14,15 @@ else
     git clone https://github.com/nnstreamer/nnstreamer-example.git Documentation/nnst-exam
 fi
 
+mkdir -p tmp_doc
+cp *.md tmp_doc
+cp Documentation/*.md .
+cp Documentation/hotdoc/doc-index.md .
+
+sed -i 's+\.\./++g; s+(Documentation/+(+g' *.md
```

Comment:
This still incurs side-effect:
```
diff --git a/Documentation/how-to-write-testcase.md b/Documentation/how-to-write-testcase.md
index 8fd6917e..015fde9c 100644
--- a/Documentation/how-to-write-testcase.md
+++ b/Documentation/how-to-write-testcase.md
@@ -16,7 +16,7 @@ title: How to write Test Cases
   - The following is a template:
   ```
   #!/usr/bin/env bash
-  source ../testAPI.sh
+  source testAPI.sh
   
   CASENUMBER1=1
   CASENUMBER2=2
```

Reply:
MY bad. Thanks! Updated.